### PR TITLE
fix: Updating regular Docker Images for helm chart.

### PR DESCRIPTION
**Pull request by WaterKnight1998**
I tried to use your regular images. However, I found that regular ones are not up to date. When you try to build these images you get lot of errors.

So this pull request updates those images.

### CODE REVIEW

File: tools/helm/livy/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
-FROM java:openjdk-8-jdk
-MAINTAINER Dalitso Banda <dalitsohb@gmail.com>
+FROM waterknight/spark:2.4.5
```
Comment (mhamilton723) [inline tools/helm/livy/Dockerfile:1]: can you add the required lines to this file instead of depending on your spark2.4.5?

**Comment by mhamilton723** [comment]
@dbanda can you take a look at this?

File: tools/helm/livy/Dockerfile (hunk 1 above)
Comment (WaterKnight1998) [reply tools/helm/livy/Dockerfile:1]: What lines do you refer? @mhamilton723

Comment (WaterKnight1998) [reply tools/helm/livy/Dockerfile:1]: I think that you don't want to depend on my Image. This image is built from spark Dockerfile.

I am going to remove that and push it again. Is enought with just pushing?

Comment (WaterKnight1998) [reply tools/helm/livy/Dockerfile]: @mhamilton723 Done!

File: tools/helm/livy/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,55 @@
...
+ADD jars /jars
+
+# if numpy is installed on a driver it needs to be installed on all
+# workers, so install it everywhere
+RUN apt-get update && \
+    apt install -y python3-pip && \
+    pip3 install numpy && \
+    pip3 install matplotlib && \
+    pip3 install pandas==0.24.1 && \
+    pip3 install fbprophet && \
```
[the remark this answers is not in the document]
Comment (WaterKnight1998) [reply tools/helm/livy/Dockerfile:41]: @dbanda Okey, I will delete it and push again. 5 minutes!

**Comment by WaterKnight1998** [comment]
@dbanda and @mhamilton723 I removed fbprophet, it was there because I built those images for a personal project. 

Should I add my name as maintainer?

**Comment by dbanda** [comment]
@WaterKnight1998 to keep things simple and secure we keep the maintainers as members of the mmlspark team.

**Comment by WaterKnight1998** [comment]
> @WaterKnight1998 to keep things simple and secure we keep the maintainers as members of the mmlspark team.

Solved @dbanda 

@dbanda and @mhamilton723 it is ready for merge!!

**Comment by WaterKnight1998** [comment]
@dbanda & @mhamilton723 have you checked it?

**Comment by WaterKnight1998** [comment]
@mhamilton723 & @dbanda did you forgive it?

**Comment by dbanda** [comment]
LGTM

**Comment by mhamilton723** [comment]
Thanks @WaterKnight1998  ill start the build

**Comment by WaterKnight1998** [comment]
> 
> 
> Thanks @WaterKnight1998 ill start the build

Thank you!!

**Comment by WaterKnight1998** [comment]
@mhamilton723 @dbanda did you forgive it? Could you merge it, pelase? It has been more than a month...

**Comment by mhamilton723** [comment]
@WaterKnight1998  sorry for the delay just merged!

**Pull request merged**